schemas/JSON/manifests/v1.2.0/manifest.locale.1.2.0.json

```
@@ -35,6 +35,21 @@
           "description": "The agreement URL."
         }
       }
+    },
+    "Documentation": {
+      "type": "object",
+      "properties": {
+        "DocumentLabel": {
+          "type": [ "string", "null" ],
+          "minLength": 1,
+          "maxLength": 100,
+          "description": "The description of the documentation for providing software guides such as manuals and troubleshooting URLs."
+        },
+        "DocumentUrl": {
+          "$ref": "#/definitions/Url",

```

Documents should always have a URL
```suggestion
          "allOf": [
            {"$ref":"#/definitions/Url"},
            {"not" : {"type":"null"}}
          ],
```